app/client/cli/cmd.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 const (
-	cliExecutableName = "client"
+	cliExecutableName = "p1"
```

Comment:
A lot of infra relies on this path indirectly and any change will need to be thoroughly vetted through that process. I'm not opposed to a name change on good grounds, but I'm not understanding how this is worthwhile. In the past, `pocket` has been avoided because it wasn't descriptive in context, so `client` could certainly be improved upon. `p1` is similarly vague, I would argue. Naming is a hard problem 😅

Reply:
## Re validation

Good point. @h5law Can you sanity check (manual with localnet, make test_e2e, etc) just to make sure it passes all the checkboxes we have in the new PR template.

@dylanlott Do you think that would be enough?

## Re Naming

`p1` was actually selected very carefully and what we use in `build.mk`. I believe we should continue using it

![Screenshot 2023-05-17 at 1 49 10 PM](https://github.com/pokt-network/pocket/assets/1892194/3a7e8423-b8a0-4099-a6eb-d1fbbbed6bb3)

Reply:
Updated `e2e` tests to use the new `p1` binary name in k8s 👍🏼

Reply:
Once the PR (#737) from @okdas merges in, you can just add the `e2e-devnet-test` label like we mention in [1]

[1] https://github.com/pokt-network/pocket/wiki/Development-Readme#github-labels